src/site/content/en/react/how-amp-can-guarantee-fastness-in-your-nextjs-app/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: How AMP can guarantee fastness in your Next.js app
+authors:
+  - houssein
+subhead: |
+  Learn about the benefits and tradeoffs of supporting AMP in your Next.js app
+date: 2019-10-17
+draft: true
+codelabs: how-to-use-amp-in-nextjs
+---
+
+[AMP](https://amp.dev) (Accelerated Mobile Pages) is a web component framework that guarantees
+fast page loads. [Next.js][intro] has built-in support for AMP.
+
+## What will you learn?
+
+This guide first briefly describes [how AMP guarantees fast page loads](#overview), then
+explains [the different ways you can support AMP in a Next.js app](#strategies),
+then helps you [decide which approach is best for you](#guidance).
+
+The intended audience for this guide is a web developer who has decided to use Next.js but is
+unsure of whether to support AMP. 
+
+{% Aside %}
+  This guide was not written for web developers who have decided to use AMP but are unsure of what
+  framework to use. We will note briefly however that Next.js could be a good choice because it
+  supports [AMP server-side rendering](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/server-side-rendering/)
+  and makes it easy to serve AMP content without introducing a lot of complexity into your codebase.
+{% endAside %}
+
+## How AMP guarantees fast page loads {: #overview }
+
+AMP has two main strategies for guaranteeing fastness:
+
+* **AMP HTML**: A restricted form of HTML that makes certain optimizations mandatory and prohibits
+  architectural patterns that lead to slowness. See [How AMP works][how] for a high-level
+  overview of the optimizations and restrictions.
+* **AMP Cache**: A content cache used by some search engines, such as Google and Bing, that uses
+  [prerendering] to speed up page loads. See [Why AMP Caches exist][why] to learn more about the
+  benefits and tradeoffs of the caches and [How does my AMP page get cached?][how] to understand
+  how your AMP pages get into the caches.
+
+## How you can use AMP in Next.js {: #strategies }
+
+There are two ways to use AMP in Next.js:
+
+* The **Hybrid AMP** approach lets you create an accompanying AMP version of any
+  Next.js page.
+* The **AMP-only** approach lets you make AMP the only option for a page.
+
+Although Next.js is usually thought of as a React framework, it's important to understand that
+when you use Next.js to serve AMP pages, you can no longer run React components client-side because
+React components are not valid AMP components. In other words, Next.js is no longer a React
+framework but rather a server-side templating engine for generating AMP pages.
```

Comment:
I think this paragraph captures Alan's conceptual idea that when you use AMP in Next.js, you should stop thinking about Next.js as a React framework but rather an AMP templating engine.